backend/src/StamAcasa.JobScheduler/appsettings.json

```diff
@@ -23,6 +23,19 @@
     "StamAcasa.JobScheduler.Jobs.SendAssessmentReminderJob": "0 9,18 * * *",
 
     // Job that sends result notifications and runs daily at 07:00.
-    "StamAcasa.JobScheduler.Jobs.SendResultsNotificationJob": "0 7 * * *"
+    "StamAcasa.JobScheduler.Jobs.SendResultsNotificationJob": "0 20 * * *"
+  },
+  "CountyEmailDistribution": {
+    "sender": "code4contact@code4.ro",
+    "templateFile": "whatever.html",
```

Comment:
I don't think we need the template here.
We will use templates based on the enumeration we are using in the Email Service.